src/state/apis/limit-orders/limitOrderApi.ts

```diff
@@ -0,0 +1,108 @@
+import { createApi } from '@reduxjs/toolkit/query'
+import type { ChainId } from '@shapeshiftoss/caip'
+import type { SwapperConfig } from '@shapeshiftoss/swapper'
+import { getCowswapNetwork } from '@shapeshiftoss/swapper'
+import { cowService } from '@shapeshiftoss/swapper/src/swappers/CowSwapper/utils/cowService'
+
+import { BASE_RTK_CREATE_API_CONFIG } from '../const'
+import type {
+  CancelLimitOrdersRequest,
+  GetOrdersRequest,
+  LimitOrder,
+  LimitOrderRequest,
+  Order,
+  OrderStatusResponse,
+  Trade,
+} from './types'
+
+export const limitOrderApi = createApi({
+  ...BASE_RTK_CREATE_API_CONFIG,
+  reducerPath: 'limitOrderApi',
+  keepUnusedDataFor: Number.MAX_SAFE_INTEGER, // never clear, we will manage this
+  tagTypes: ['LimitOrder'],
+  endpoints: build => ({
+    getLimitOrder: build.mutation<
+      LimitOrder,
+      { payload: LimitOrderRequest; config: SwapperConfig }
+    >({
+      queryFn: async ({ payload, config }) => {
+        const baseUrl = config.REACT_APP_COWSWAP_BASE_URL
+        const maybeNetwork = getCowswapNetwork(payload.chainId)
+        if (maybeNetwork.isErr()) throw maybeNetwork.unwrapErr()
+        const network = maybeNetwork.unwrap()
+        const maybeResult = await cowService.post<LimitOrder>(
+          `${baseUrl}/${network}/api/v1/orders/`,
+          payload,
+        )
+        if (maybeResult.isErr()) throw maybeResult.unwrapErr()
+        const { data: order } = maybeResult.unwrap()
+        return { data: order }
```

Comment:
[preferably blocking]: These monads are pointless in a non-monadic context, can we use axios without monadic service here like in thorchain swapper `endpoints.ts`?

![image](https://github.com/user-attachments/assets/144cc019-908f-4384-835b-37c1d8e5709c)

Reply:
Ignore the promise.all, referring to rawdogging axios

Reply:
Can do, rawdogged in https://github.com/shapeshift/web/pull/7906/commits/007aadb83e2a7e9ae997fb38a97c2be9d34f26b7